modules/apigee_edge_apidocs/README.md

```diff
@@ -0,0 +1,37 @@
+# API Docs for Apigee Edge
+
+A module to render OpenAPI specs as documentation to your API developers.
+
+## Overview
+
+When you enable this module, it creates a new Drupal entity in your system named
+"API Doc". You can add new API Docs under Structure > API Docs in the admin menu.
+
+Once added, the API name and description for each API Doc will be displayed in the
+"APIs" menu item on the site to all visitors.
+
+The OpenAPI spec by default is shown on the API Doc detail page by default.
+To render the OpenAPI spec using Swagger UI:
+
+1. Download an enable the [Swagger UI Field Formatter](https://www.drupal.org/project/swagger_ui_formatter) module
```

Comment:
TODO: The swagger UI library also has to be installed to `libraries/swagger_ui` (missing step).